PacketDefinitions420/PacketNotifier.cs

```diff
@@ -721,23 +721,71 @@ public void NotifyLeaveVision(IGameObject o, TeamId team)
             }
         }
 
-        public void NotifyEnterVision(IGameObject o, TeamId team)
+        // TODO: Fix IsAnythingBetween randomly losing vision often, which ends up spamming this as well
+        public void NotifyEnterVisibilityClient(IAttackableUnit u, TeamId team, int userId = 0)
```

Comment:
Related implementation: #832

Reply:
Not sure if you want me to change something, the implementation linked has many features that my PR lacks, such as SyncId and MovementData, thus my implementation can't change much.

Reply:
I just wanted to relate this in GitHub (so I'll have a link).